### PR TITLE
Hc metadata parent

### PR DESCRIPTION
**Pull request by FelipeRibeiroLabs**

### CODE REVIEW

File: contracts/HybridCustody.cdc [hunk 1]
```diff
@@ -271,6 +271,18 @@ pub contract HybridCustody {
         // capabilities.
         pub var filter: Capability<&{CapabilityFilter.Filter}>?
 
+        // display is its own field on the ChildAccount resource because only the parent should be able to set this
+        // field.
```
Comment (austinkline) [inline contracts/HybridCustody.cdc:275]: This comment isn't really accurate anymore, maybe something like:

```suggestion
        // display metadata for a child account exists on its parent
```

File: contracts/HybridCustody.cdc [hunk 2]
```diff
@@ -271,6 +271,18 @@ pub contract HybridCustody {
...
+        pub let childAccountDisplays: {Address: MetadataViews.Display}
+
+        pub fun setChildAccountDisplay(address: Address, _ d: MetadataViews.Display) {
```
Comment (austinkline) [inline contracts/HybridCustody.cdc:278]: We either need a way for a display to be removed on this method (let it be an optional display and remove from the dictionary if nil, or we should add a new method

**Review by austinkline — CHANGES_REQUESTED** [review]
Two small pieces, looks good otherwise so far!

1. Comment cleanup
2. How should a parent remove a display on a child?

**Comment by sisyphusSmiling** [comment]
> How should a parent remove a display on a child?

I think setting as `nil` like you mentioned is a lightweight method for deletion, and an upsert if not nil makes sense to me for this use case.

**Review by austinkline — APPROVED** [review]

**Pull request merged**